.eslintrc.js

```diff
@@ -3,4 +3,15 @@ module.exports = {
   env: {
     node: true,
   },
+  rules: {
```

Comment:
Turning rules off is not OK.

Reply:
I completely agree, hence the commit message saying to drop the commit that does this in the merge.